src/Fixer/Symfony/MethodSeparationFixer.php

```diff
@@ -38,7 +38,7 @@ public function getDescription()
      */
     public function getPriority()
     {
-        // Must run before braces and indentation fixers because this fixer
+        // Must run before braces and no_tabs fixers because this fixer
```

Comment:
All name-changes was made automatically. That is why there is some false-positive changes that should be reverted.
Some comments in `getPriority` uses Fixer names, some rule names. While unification is good, it is not scope of this PR.